File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.java
class Solution {
    public int minMaxDifference(int num) {
        String max = Integer.toString(num);
        String min = Integer.toString(num);
        for(char ch : max.toCharArray()){
            if(ch != '9'){
                max = max.replace(ch,'9');
                break;
            }
        }
        min = min.replace(min.charAt(0),'0');
        return Integer.parseInt(max) - Integer.parseInt(min);
    }
}

File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.py
class Solution:
    def minMaxDifference(self, num: int) -> int:
        max = str(num)
        for ch in max:
            if ch != '9':
                max = max.replace(ch,'9')
                break
        min = str(num)
        min = min.replace(min[0],'0')
        return int(max) - int(min)
        